File: olymp.tj/National Olympiads/9-10_region_2026(round 2)/D.cpp
#include <bits/stdc++.h>
#define ll long long
using namespace std;

int main() {
    string s;
    cin >> s;
    int cnt=0;
    for (char &c:s) cnt+=isupper(c);
    if (cnt==s.size() or (cnt==s.size()-1 and islower(s[0]))){
        for (auto &c:s){
            if (islower(c)) c = toupper(c);
            else c = tolower(c);
        }
    }
    cout << s;
}

File: olymp.tj/National Olympiads/9-10_region_2026(round 2)/D.py
s = input()
cnt = 0
for c in s:
    if c.isupper():
        cnt += 1

if cnt == len(s) or (cnt == len(s)-1 and s[0].islower()):
    new_s = ''
    for c in s:
        if c.isupper():
            new_s += c.lower()
        else:
            new_s += c.upper()
    s = new_s

print(s)
